### ISSUE TITLE
Incorrect F1 scores

### ISSUE THREAD

**Issue by david-waterworth** [semantic-health/allennlp-multi-label#176]
You're calculating the F-score using logits but passing a default threshold of 0.5

I'm fairly sure you should be passing probabilities instead or set the threshold to 0.0 - as far as I can see `F1MultiLabelMeasure` doesn't convert logits to probabilities, it simply compares the predictions with the threshold.
